### DISCUSSION TITLE
Hey Everyone !

### DISCUSSION THREAD

**Discussion by salil-gtm** [OpenMined/courses#25] [Introduce Yourself]
# :handshake: 
I'm currently working as a Data Scientist from Bangalore. 
Feel free to connect with me on [LinkedIn](https://www.linkedin.com/in/salil-gautam/) !

**Comment by yemikifouly** [comment]
Hello Salil! 🙂 You work at Societe Generale? There is an office of them not far from my place and I am on the other side of the world. 😀 Anyway, welcome to the discussion board. I really hope you enjoy the course!

**Reply by salil-gtm** [reply]
Yes, I do ! Small World :earth_asia:
